README.md

```diff
@@ -1,153 +1,146 @@
 * [Private Repositories](#private-repositories)
 * [Pull Secrets](#pull-secrets)
 * [IPV4 enforcement](#ipv4-enforcement)
 
+>Note: Please see the release notes for what's new in the latest release.
 
-#### Prerequisites:
-* A minimum of 3 nodes which support the following [requirements][]
-* A kubernetes version of 1.8 or higher
-* For service broker - a k8s distribution that supports service catalog (see also: [service-catalog][])
-* Access to DockerHub, RedHat Container Catalog or a private repository that can serve the required images
-> Note: For RHEL based images and/or deployments on OpenShift, please use redis-enterprise-cluster_rhel.yaml and operator_rhel.yaml.
-For Service Broker, please see examples/with_service_broker_rhel.yaml. RedHat certified images are available on: https://access.redhat.com/containers/#/product/71f6d1bb3408bd0d
-
-The following are the images and tags for this release:
-
-Redis Enterprise    -   `redislabs/redis:5.4.6-18` or `redislabs/redis:5.4.6-18.rhel7-openshift`
-
-Operator            -   `redislabs/operator:5.4.6-1186` or `redislabs/operator:5.4.6-1186b.rhel7`
+## Additional Documentation
 
-Services Rigger     -   `redislabs/k8s-controller:5.4.6-1186` or `redislabs/k8s-controller:5.4.6-1186b.rhel7`
+- [Advanced Topics](docs/topics.md)
+- [Resource Specification Reference](docs/operator.md)
 
-Service Broker      -   `redislabs/service-broker:78_4b9b17f` or `redislabs/service-broker:78_4b9b17f.rhel7`
+## Quickstart Guide
 
+### Prerequisites
 
+- A minimum of 3 nodes which support the following requirements
+- A Kubernetes cluster (server) version of 1.8 or higher
+- A Kubernetes client (kubectl) with a matching version. For OpenShift, an OpenShift client (oc).
+- For service broker - a k8s distribution that supports service catalog (see also: service-catalog)
+- Access to DockerHub, RedHat Container Catalog or a private repository that can serve the required images
 
-#### Deployment:
-Clone (or download) this repository, which contains the deployment files:
-```
-git clone https://github.com/RedisLabs/redis-enterprise-k8s-docs.git
-```
-
-1) Create a namespace / project.
+For Service Broker, please see examples/with_service_broker_rhel.yaml. RedHat certified images are available on: https://access.redhat.com/containers/#/product/71f6d1bb3408bd0d
 
-    For non-OpenShift deployments, create a new namespace:
+The following are the images and tags for this release:
 
-    ```
-    kubectl create namespace demo
-    ```
+Redis Enterprise    -   `redislabs/redis:5.4.10-22` or `redislabs/redis:5.4.6-22b.rhel7-openshift`
 
-    For OpenShift deployments, create a new project (you can substitute `oc` for `kubectl` in the rest of these instructions):
+Operator            -   `redislabs/operator:5.4.10-8` or `redislabs/operator:5.4.6-8.rhel7`
 
-    ```
-    oc new-project my-project
-    ```
+Services Rigger     -   `redislabs/k8s-controller:5.4.10-8` or `redislabs/k8s-controller:5.4.10-8b.rhel7`
```

Comment:
5.4.10-8**b**?

Reply:
it is the image on RedHat site

Reply:
8b is on RedHat connect which we needed to retag due to the labels issue